boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -22,6 +22,7 @@
 		zephyr,code-partition = &cpuapp_slot0_partition;
 		zephyr,flash = &mram1x;
 		zephyr,sram = &cpuapp_data;
+		zephyr,shell-uart = &uart136;
```

Comment:
Why only for `cpuapp`?

Reply:
I was just using CPU APP :), I can extend to other processors too, but I can't test them.